common/event_modifiers/wc_event_modifiers.txt

```diff
@@ -1701,4 +1701,9 @@ writing_whispers = {
 	}
 
 	monthly_character_piety = 0.25
+}
+twilight_architecture = {
```

Comment:
Not cleared when you change religion.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77834681-cccc6300-715f-11ea-8464-b7a96e916a68.png)

</details>